run_Wochenende_SLURM.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+## A SLURM sbatch script which is part of the Wochenende Pipeline
+## Supply the FASTA input as arg1, bash run_Wochenende_slurm.sh in_R1.fastq
+
+# set partition
+#SBATCH -p normal
+
+# set run on x MB node only
+#SBATCH --mem 40000
+
+# set run x cpus
+#SBATCH --cpus-per-task 12
+
+# set max wallclock time
+# SBATCH --time=47:00:00
+
+# set name of job
+#SBATCH --job-name=Wochenende
+
+# mail alert at start, end and abortion of execution
+#SBATCH --mail-type=ALL
+#SBATCH --mail-user=<mail_address_for_slurm>
+
+echo "Input file: " $1
+fastq=$1
+
+echo "output dir: " $2
+outDir=$2
+
+# Add miniconda3 to PATH
+. /mnt/ngsnfs/tools/miniconda3/etc/profile.d/conda.sh
+
+# Activate env on cluster node
+conda activate wochenende >> /dev/null
+
+
+### IMPORTANT
+# Remember to check specified a) refseq b) threads c) adapters
+##########
+
+# All options
+# --threads 12
+#--longread (implies aligner is not bwa-mem, no prinseq, no duplicate removal)
+#--aligner bwamem
+#--aligner minimap2
+#--aligner ngmlr
+#--no_abra
```

Comment:
I just updated this - pending your review - with the nextera removal so you'll have to update this file.